dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2249,7 +2251,7 @@ def test_graph_size_pyarrow(tmpdir, engine):
     ddf1.to_parquet(fn, engine=engine)
     ddf2 = dd.read_parquet(fn, engine=engine)
 
-    assert len(pickle.dumps(ddf2.__dask_graph__())) < 10000
+    assert len(pickle.dumps(ddf2.__dask_graph__())) < 11000
```

Comment:
I'm probably missing something simple, but what prompted this change?

Reply:
Sorry - Shouldn't be necessary. This was a left-over change from an earlier iteration where the graph was slightly larger.